tut_rule/tut.bzl

```diff
@@ -1,21 +1,29 @@
 load("//scala:scala.bzl", "scala_binary")
 load("//scala:scala_cross_version.bzl", "scala_mvn_artifact")
 
-def tut_repositories():
+def tut_repositories(scala_version = "2.11.11"):
+  major_version_underscore = scala_version[:scala_version.find(".", 2)].replace(
+      ".", "_")
   native.maven_server(
       name = "tut_repositories_maven_server",
       url = "https://dl.bintray.com/tpolecat/maven/",
   )
 
   native.maven_jar(
-      name = "io_bazel_rules_scala_org_tpolecat_tut_core",
+      name = "io_bazel_rules_scala_org_tpolecat_tut_core_2_11",
       artifact = scala_mvn_artifact("org.tpolecat:tut-core:0.4.8"),
```

Comment:
I wish we could update this—0.4.8 is over a year and a half old, which is a looooong time in the life of a young-ish project like this.